Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2_test.py

```diff
@@ -0,0 +1,349 @@
+import json
+import pytest
+import io
+from CommonServerPython import Common, DemistoException, ExecutionMetrics
+from typing import *
```

Comment:
not a good practice to do this in general - better providing what you need explicitly, because for example, if `typing` had a method `client` you will override it in line 14 which might corrupt the code.